management/management.py

```diff
@@ -176,6 +176,13 @@ def restart_optimizations(optimizations, client):
             client.modify_tasks(operation='restart', base_result=opt.id)
```

Comment:
Are we sure that regenerated tasks are automatically eligible for submission? Should we try to restart after regen?

Reply:
I did an ad-hoc test with the old Fragmenter dataset we are trying to get clear of INCOMPLETEs. I'm not sure it had any effect. Posed a more targeted question in slack.

Reply:
Looks like it cleared from Friday's discussion!